ecs_deploy/ecs.py

```diff
@@ -329,6 +377,17 @@ def _get_environment_diffs(container, env, old_env):
                 diffs.append(message)
         return diffs
 
+    @staticmethod
+    def _get_secrets_diffs(container, secrets, old_secrets):
+        msg = u'Changed secrets "%s" of container "%s" to: "%s"'
```

Comment:
As this is about a single secret, I think it should be singular ("Changed secret ...")

Reply:
Yup agreed.